package.json

```diff
@@ -5,7 +5,7 @@
     "bootstrap-no-i": "asini bootstrap",
     "test": "asini run test",
     "clean": "rimraf asini-debug.log && asini run clean",
-    "nuke": "asini clean && rm -r node_modules",
+    "nuke": "npm run clean && asini clean",
```

Comment:
I think we still actually need the `rm -r node_modules` here, since the root isn't a package that's managed by Asini.